content/plugins/provide-plugin.md

```diff
@@ -38,3 +41,11 @@ new webpack.ProvidePlugin({
   'window.jQuery': 'jquery'
 })
 ```
+
+### Use Vue.js(ES Modules)
+
+```javascript
+new webpack.ProvidePlugin({
+  Vue: ['vue/dist/vue.esm.js', 'default'] // v2.2.6
```

Comment:
Thank you for your confirmation.
I'll re-update the document after webpack/webpack#4729 is solved.

> If you want to keep the Vue section, please remove the "(ES Modules)" from the caption and the version comment.

Okay.
if it is determined that `ProvidePlugin` behaves like `require` as is, I'll add the Vue section  with displaying a warning box.